js/open-photo.js

```diff
@@ -1,27 +1,24 @@
-import { photos } from './mock.js';
 import { isEscapeKey } from './util.js';
+import { addComments, resetComments } from './render-comments.js';
 
-const picturesBlock = document.querySelector('.pictures');
 const bigPicture = document.querySelector('.big-picture');
-const closeButton = document.querySelector('.big-picture__cancel');
-const commentTemplate = document.querySelector('#comment').content.querySelector('.social__comment');
 const mainPicture = bigPicture.querySelector('.big-picture__img img');
 const likesCount = bigPicture.querySelector('.likes-count');
-const commentsBlock = bigPicture.querySelector('.social__comments');
 const commentsCount = bigPicture.querySelector('.comments-count');
-const commentsCurrentCount = bigPicture.querySelector('.comments-current-count');
 const pictureCaption = bigPicture.querySelector('.social__caption');
+const closeButton = document.querySelector('.big-picture__cancel');
 const commentsButton = bigPicture.querySelector('.comments-loader');
 
-let addComments;
+let comments;
```

Comment:
Это функция. А сейчас по названию непонятно
```suggestion
let commentsButtonClickHandler;
```